plugins/scaffolder/src/components/fields/RepoUrlPicker/GithubRepoPicker.tsx

```diff
@@ -42,15 +44,31 @@ export const GithubRepoPicker = (props: {
         error={rawErrors?.length > 0 && !owner}
       >
         {allowedOwners?.length ? (
-          <Select
-            native
-            label="Owner Available"
-            onChange={s =>
-              onChange({ owner: String(Array.isArray(s) ? s[0] : s) })
-            }
-            disabled={allowedOwners.length === 1}
-            selected={owner}
-            items={ownerItems}
+          <Autocomplete
+          aria-label="Owner Available"
+          options={ownerItems}
+          getOptionLabel={(owners) => owners.label || "error getting label"}
+          disabled={allowedOwners.length === 1}
+          data-testid="select"
+          freeSolo
+          onChange={selected =>
+            onChange({
+              owner: String(Array.isArray(selected) ? selected[0] : selected),
+            })
+          } 
+          tabIndex={0}
+          renderInput={params => (
+              <TextField
+              {...params}
+              placeholder="Owner Available"
```

Comment:
Think that this should just say the relevant thing you're filtering. Like `Select an Owner`

Reply:
Thank you @benjdlambert . Let me make the changes. Thanks for the support thus far.. :)